### PR TITLE
[REF] sms marketing: restructure/add cards/remove essen.

### PR DESCRIPTION
**Pull request by ksc-odoo**
In an effort to clean up the look (and feel) of the Marketing scope of applications, I am creating this PR to remove the 'essentials' and 'Pricing and FAQ' subsections , add intro content to the main SMS Marketing element on the toctree, add some linked cards to the main SMS documentation page (like Email Marketing, Events, etc.). 

Not only does this help everything appear uniform, but adds a much-needed level of organization and eases the navigability of the documentation.

And, just like the Events doc, I am *also* adding individual sections (w/ accompanying explanations and links) to the other pages in the scope.

**I PLAN ON UPDATING ALL CONTENT **AFTER** THIS PR (which sets up the new structure and layout)**
This PR is **purely** about structure and the addition of cards

PROJECT TASK: https://www.odoo.com/odoo/project/3835/tasks/4063869?cids=3

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey @Felicious -- this is ready for a quick Peer Review from you (whenever you get a chance). Thanks! 👍 🙏

**Review by Felicious — CHANGES_REQUESTED** [review]
It's great that you're doing the restructure for these SMS Marketing docs, @ksc-odoo !

While the rewrite is good, I found a couple outdated screenshots and opportunities to expand on existing fields. After you address those, tag me again and I'll be happy to look at the PR again!

**Review by Felicious — APPROVED** [review]
@ksc-odoo just let me know that this PR is purely just moving content around! Whoops! Since there are no build issues and the structure of the content looks good, approving this now :D

**Comment by ksc-odoo** [comment]
Thank you SO MUCH for the insightful (and super helpful) feedback, @Felicious -- and, yes, as we discussed, this is purely a restructure doc -- and I will be tackling updates for the individual docs in the very near future. Also, thanks to your help, I made sure to add the necessary redirect to this PR, so everything *should* be good to go. 

@StraubCreative / @samueljlieber this is ready for a Final/Tech review, whenever you get a chance. Thanks! 🙏

File: content/applications/marketing/sms_marketing.rst [hunk 1]
```diff
@@ -1,17 +1,83 @@
...
+
+   .. card:: SMS analysis
+      :target: sms_marketing/sms_analysis
+
+      Examine detailed reporting metrics related to every aspect of sent SMS messages.
+
+   .. card:: SMS marketing campaigns
+      :target: sms_marketing/marketing_campaigns
+
+      Discover how to create and customize SMS marketing campaigns for any occassion with Odoo.
```
Comment (samueljlieber) [inline content/applications/marketing/sms_marketing.rst:34]: Typo
```suggestion
      Discover how to create and customize SMS marketing campaigns for any occasion with Odoo.
```

File: redirects/17.0.txt [hunk 2]
```diff
@@ -64,6 +64,7 @@ applications/marketing/marketing_automation/getting_started/target_audience.rst
 applications/marketing/marketing_automation/getting_started/testing_running.rst applications/marketing/marketing_automation/testing_running.rst           # getting_started/* --> marketing_automation/*
 applications/marketing/marketing_automation/getting_started/workflow_activities.rst applications/marketing/marketing_automation/workflow_activities.rst   # getting_started/* --> marketing_automation/*
 applications/marketing/surveys/time_random.rst applications/marketing/surveys/questions.rst   # time_random --> questions
+applications/marketing/sms_marketing/essentials.rst applications/marketing/sms_marketing.rst 								  # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
```
Comment (samueljlieber) [inline redirects/17.0.txt:67]: Need to add redirects for all moved and deleted files :)

Also please only use spaces, never tabs, for spacing and indenting in `.rst` and redirect `.txt` files :)
```suggestion
applications/marketing/sms_marketing/essentials.rst applications/marketing/sms_marketing.rst                                                              # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
applications/marketing/sms_marketing/essentials/sms_essentials.rst applications/marketing/sms_marketing.rst                                               # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
applications/marketing/sms_marketing/essentials/sms_campaign_settings.rst applications/marketing/sms_marketing.rst                                        # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst applications/marketing/sms_marketing/mailing_lists_blacklists.rst            # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
applications/marketing/sms_marketing/pricing.rst applications/marketing/sms_marketing/pricing_and_faq.rst                                                 # /marketing/sms_marketing/pricing --> /marketing/sms_marketing
applications/marketing/sms_marketing/pricing/pricing_and_faq.rst applications/marketing/sms_marketing/pricing_and_faq.rst                                 # /marketing/sms_marketing/pricing --> /marketing/sms_marketing
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Nice work on this restructure, I only have a couple comments! I know these are a big lift 💪

Also please update the commit message to be more succinct: 
`git commit --amend`
```
[REF] sms marketing: remove essentials and pricing
```
Requesting changes so I can  double check redirects, tag me for one more quick look!

Thank you!

**Comment by ksc-odoo** [comment]
I made those suggested adjustments, and *believe* 🤞 this one is now ready for another look from you, whenever you get a chance. Thanks for all the awesome help! 🙏 👍

**Comment by ksc-odoo** [comment]
@samueljlieber just noticed I got a build error on this one, but I'm not quite sure how to resolve it. Any expert guidance or tips would be greatly appreciated (as always 🙏 )

File: redirects/17.0.txt [hunk 3]
```diff
@@ -67,9 +67,16 @@ applications/marketing/marketing_automation/getting_started/target_audience.rst
 applications/marketing/marketing_automation/getting_started/testing_running.rst applications/marketing/marketing_automation/testing_running.rst           # getting_started/* --> marketing_automation/*
 applications/marketing/marketing_automation/getting_started/workflow_activities.rst applications/marketing/marketing_automation/workflow_activities.rst   # getting_started/* --> marketing_automation/*
 applications/marketing/surveys/time_random.rst applications/marketing/surveys/questions.rst   # time_random --> questions
+<<<<<<< HEAD
```
Comment (samueljlieber) [inline redirects/17.0.txt:70]: Somehow this ended up in the redirect file, likely left over from resolving a conflict :)
```suggestion
```

**Review by samueljlieber — APPROVED** [review]
Hey @ksc-odoo, these redirects look good, just need to remove this line left over from a conflict. 

LMK if you have any questions! Thanks!
...
@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**